relays/ethereum/src/exchange_loop.rs

```diff
@@ -92,6 +95,19 @@ pub fn run<P: TransactionProofPipeline>(
 		let mut state = storage.state();
 		let mut current_finalized_block = None;
 
+		let mut metrics_global = GlobalMetrics::new();
```

Comment:
I feel like this kind of code + `expose_metrics` is duplicated in every relay loop. Is there a way we could wrap it into a nicer API? Ideally it would be 2-3 lines at most to add new metrics to another `*_loop.rs` file imho (init + get_state + update).